File: Assignment2/run/InhousePatient1.cs
namespace Hospital;
public class  InhousePatient1 : Patient
{
	public double discount {get;set;}
	
	public InhousePatient1 (double pid, String name, int btype, double noofdays,double discount): base (pid,  name,  btype,  noofdays){
	this.discount = discount;
	 }
	public  override  double BillingAmount()
	{
		double amount = base.BillingAmount();
		return amount<5000 ? (amount*0.95) : (amount - amount*(discount/100));
	}
	
}

File: 2_JAVA/Assignment 3/InhousePatient1.java
package Hospital;
public class InhousePatient1 extends Patient{
	public InhousePatient1(double pid,String name,int btype,double days,double disc){
		super(pid,name,btype,days);
		discount = disc;
	}
	public double getdiscount (){
		return discount;
	}
	public void setdiscount (double disc){
		discount = disc;
	}
	public double getbillingamount(){
		double amount = super.getbillingamount();
		return amount<5000 ? (amount*0.95) : (amount - amount*(discount/100));
	}
	private double discount;
}
